src/test/java/org/numenta/nupic/network/LayerTest.java

```diff
@@ -307,7 +311,7 @@ public void testHalt() {
         Parameters p = NetworkTestHarness.getParameters().copy();
         p = p.union(NetworkTestHarness.getHotGymTestEncoderParams());
         p.set(KEY.RANDOM, new MersenneTwister(42));
-        p.set(KEY.AUTO_CLASSIFY, Boolean.TRUE);
+        p.set(KEY.AUTO_CLASSIFY, false);
```

Comment:
@Hopding  Am I looking at the current code? It looks like this wasn't changed back to true?